doc/releases/release-notes-4.0.rst

```diff
@@ -266,6 +319,17 @@ Boards & SoC Support
 
   * Added Thingy53 forwarding of network core pins to network core for SPI peripheral (disabled
     by default) including pin mappings.
+  * Added uart, flexio pwm, flexio spi, watchdog, flash, rtc, i2c, lpspi, edma, gpio, acmp, adc and lptmr support
+    to NXP ``frdm_ke17z`` and ``frdm_ke17z512``
+  * Enabled support for MCUmgr on NXP boards
+  * Added documentation for SEGGER SystemView support for all NXP boards with an SoC supporting ECC
```

Comment:
do we need to document that we added documentation

Reply:
It is one addition that wasn't there before, it contains useful information

Reply:
I agree, please remove it. It doesn't make sense to get to this level of detail. Again, please take inspiration from other contributions already made to the release notes or from previous contributions such as https://github.com/zephyrproject-rtos/zephyr/pull/69100/files.
I by no means want to minimize NXP's contributions to the release but the level of detail in this PR is really not helping keep the release notes easy to read, and I'm afraid it's a bit counterproductive that for wanting to highlight too much you will in fact highlight nothing for the reader will just be deterred.